src/index.ts

```diff
@@ -1,25 +1,47 @@
-import {processData} from "./services";
 import fs from "fs";
 import path from "path";
-import {createXlSXfile} from "./utils/xlsx";
-
-if (process.env.NODE_ENV === 'development') {
-    require('dotenv').config()
+import moment from "moment";
+import {
+  generateXLSXOrderData,
+  fetchOrderDataRecursion
+} from "./services/order";
+import { createXlSXfile } from "./utils/xlsx";
+
+if (process.env.NODE_ENV === "development") {
+  require("dotenv").config();
 }
 
-(async() => {
-    const dir = path.join(__dirname, `/exportData/`)
-
-    if (!fs.existsSync(dir)) {
-        await fs.mkdirSync(dir, {recursive: false})
-    }
-
-     const data = await processData();
-    createXlSXfile({data, dirName: 'src/exportData'})
+const DIR_NAME = process.env.DIR_NAME || "exports";
+
+(async () => {
+  console.log("\n\n\n");
+  if (!fs.existsSync(DIR_NAME)) {
+    await fs.mkdirSync(DIR_NAME, { recursive: false });
+  }
+
+  const since = moment()
+    .add(-10, "days")
```

Comment:
Why don't use `substract` instead? [Check it here](https://momentjs.com/docs/#/manipulating/subtract/).

Reply:
Before version 2.8.0, the moment#subtract(String, Number) syntax was also supported. It has been deprecated in favor of moment#subtract(Number, String).